frontend/src/metabase/styled-components/containers/GlobalStyles/GlobalStyles.tsx

```diff
@@ -15,11 +15,11 @@ export const GlobalStyles = (): JSX.Element => {
 
   const styles = css`
     :root {
-      --default-font-family: "${font}";
-      --color-brand: ${color("brand")};
-      --color-brand-alpha-04: ${alpha("brand", 0.04)};
-      --color-brand-alpha-88: ${alpha("brand", 0.88)};
-      --color-focus: ${color("focus")};
+      --mb-default-font-family: "${font}";
```

Comment:
There are basically 2 files we need to make sure they're in sync
1. `GlobalStyles.tsx` -> This is used for our main app
2. `SdkContentWrapper.tsx` -> This is used for Embedding SDK

They're mostly containing the same content with some small discrepancies

P.S. There is another file for storybook `.storybook/preview-head.html` since these are the CSS custom properties exposed in HTML we need to make sure storybook has access to these properties too.